src/state/GlobalStore.js

```diff
@@ -19,6 +19,7 @@ class GlobalStore {
       visitedPages: [],
       totalQAsByInvestigation: null,
       totalQAsByPage: null,
+      questionNumbersByPage: null,
```

Comment:
So this was the big thing.  You had this guy sitting out here going unused and it got me to thinking, yea, this is definitely where these page numbers should live, and they should be indexed by page Id, let's do that (instead of your kinda shoehorning the question numbers in amongst the totals and qa/page progress, which was also indexed by page id, so was a super convenient place to store this question numbering stuff, but it didn't really "belong").